compiler/util/clangUtil.cpp

```diff
@@ -994,13 +1004,16 @@ void finishCodegenLLVM() {
   info->FPM_postgen = NULL;
 
   // Now finish any Clang code generation.
-  cleanupClang(info);
+  finishClang(info);
+
+  if(debug_info)debug_info->finalize();
```

Comment:
If you do that the obvious way, it looks like you will call info->cgBuilder->Release() twice, which is presumably a problem, so you'll have to adjust the code to avoid that.